### PR TITLE
ECOM-5660 Add logo image url override for organizations

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
@edx/ecommerce

### CODE REVIEW

**Comment by clintonb** [comment]
Why is this necessary? Organization data currently comes from Drupal.

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -159,6 +159,14 @@ class Organization(TimeStampedModel):
     description = models.TextField(null=True, blank=True)
     homepage_url = models.URLField(max_length=255, null=True, blank=True)
     logo_image_url = models.URLField(null=True, blank=True)
+    logo_image_url_override = StdImageField(
+        upload_to=UploadToFieldNamePath(populate_from='uuid', path='media/organizations/logo_images'),
+        blank=True,
+        null=True,
+        variations={
+            'small': (200, 101),
```
Comment (schenedx) [inline course_discovery/apps/course_metadata/models.py]: Are you sure this is the only variable we need? What about also store the original?

**Review by schenedx — COMMENTED** [review]
Is there a plan to also update the dataloader so we store the original logo image for the organization from the source?

**Review by clintonb — CHANGES_REQUESTED** [review]
The issue here is that we are pulling organization data from Drupal and LMS (Organizations API).

LMS: https://courses.edx.org/api/organizations/v0/organizations/MITx/
Drupal: https://prod-edx-mktg-edit.edx.org/sites/default/files/school/image/banner/mit_logo_200x101_0.png

The change you want is to the data loaders, and determining precedence of the data. Adding an override is not correct.

**Comment by MatthewPiatetsky** [comment]
@edx/ecommerce I've updated the PR per @clintonb's comment. 
New github is confusing because you still see old comments like Simon's comment even though they are not relevant anymore.

File: course_discovery/apps/course_metadata/data_loaders/api.py [hunk 2]
```diff
@@ -50,11 +50,13 @@ def update_organization(self, body):
...
         defaults = {
             'key': key,
-            'name': body['name'],
-            'description': body['description'],
-            'logo_image_url': body['logo'],
-            'partner': self.partner,
+            'partner': self.partner
         }
+        if not self.partner.has_marketing_site:
+            defaults['name'] = body['name']
```
Comment (clintonb) [inline course_discovery/apps/course_metadata/data_loaders/api.py]:
``` python
defaults.update({
  'name': body['name'],
  ...
})
```

**Review by clintonb — APPROVED** [review]

**Review by schenedx — APPROVED** [review]
👍 
I agree. My comment clearly is a relic in the past!

**Pull request merged**